lib/ExecutionContext.js

```diff
@@ -75,7 +75,8 @@ class ExecutionContext {
       executionContextId: this._contextId,
       arguments: args.map(convertArgument.bind(this)),
       returnByValue: false,
-      awaitPromise: true
+      awaitPromise: true,
+      userGesture: true
```

Comment:
Do we need this up on 67 for the `Runtime.evaluate` call?

Reply:
Good catch. done.